src/js/menu/acquia_lift.personalize.views.js

```diff
@@ -1182,10 +1182,12 @@
       if (!this.$el.hasClass('navbar-box')) {
         return;
       }
-      this.listenTo(this.collection, 'change:isActive', this.onActiveCampaignChange);
```

Comment:
So this approach still has events firing and listeners active, etc.

Instead I would add the following to top of the initialize:
```javascript
if (!Drupal.settings.acquia_lift.allowStatusChange) {
  this.remove();
  return;
}
```

Then you can remove the changes in the render function because it is never called.